buildtest/cli/config.py

```diff
@@ -23,6 +23,9 @@ def config_cmd(args, configuration, editor, system):
     if args.config in ["view", "v"]:
         view_configuration(configuration, theme=args.theme, pager=args.pager)
 
+    if args.config in ["profile"]:
+        view_profiles(configuration, theme=args.theme, pager=args.pager)
+
```

Comment:
The name of the method is not the same as the method definition in line 158.

Reply:
fixed